src/InstantSearch.js/dist/index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Do not use @2 in production, use a complete version like x.x.x, see website for latest version:
+      https://community.algolia.com/instantsearch.js/v2/getting-started.html
+    -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.js@2/dist/instantsearch.min.css">
+	  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.js@2/dist/instantsearch-theme-algolia.min.css">
+	  <link rel="stylesheet" href="/dist/6db81cb95083dea47b35b1199432eb12.css">
+    <meta charset="utf-8">
+    <title>Algolia InstantSearch.js template</title>
+  <script src="/dist/6db81cb95083dea47b35b1199432eb12.js"></script></head>
+
+  <body>
+    <div class="container">
+      <h1>Algolia <a href="https://community.algolia.com/instantsearch.js/">InstantSearch.js</a> template</h1>
+      <p>Fully working InstantSearch.js template for creating test cases, examples and demos.</p>
+      <p><b>Start editing the index.js file</b> and see changes happening!</p>
+      <p>Available on CodeSandbox at <a href="https://codesandbox.io/s/github/algolia/instantsearch-templates/tree/master/src/InstantSearch.js">https://codesandbox.io/s/github/algolia/instantsearch-templates/tree/master/src/InstantSearch.js</a></p>
+
+      <h2>
+        Products
+      </h2>
+      <div id="searchBox"></div>
+      <div class="twoColumns">
+        <div id="brand"></div>
+        <div id="products"></div>
+      </div>
+      <div id="pagination"></div>
+    </div>
+    <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@2/dist/instantsearch.min.js"></script>
+    <script src="/dist/7780cb5818970b50c68b251f72b5893a.js"></script>
```

Comment:
Dist probably needs to be ignored

Reply:
done thanks for that, have a good conference :)